client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -16,11 +16,16 @@ const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
   return (
     <Box p={1} className={classes.authHeader}>
       <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
+      {
+        btnText === "Create account" ? 
```

Comment:
Even though it's only two states, I would still suggest refactoring auth page info such as `btnText`, `asideText` and `linkTo`  into an [enum-like object](https://www.sohamkamani.com/javascript/enums/). That way comparisons will be more robust and future changes simpler.